class RepoInternalError(Exception):
    pass

class BadRequestError(Exception):
    pass